src/test/java/org/eclipse/yasson/defaultmapping/modifiers/ClassModifiersTest.java

```diff
@@ -53,8 +53,7 @@ public void testNestedPackagePrivateParent() {
         child.id = 1;
         child.name = "SomeName";
         Assertions.shouldFail(() -> defaultJsonb.toJson(child),
-                msg -> msg.contains("Unable to serialize property 'id'") &&
```

Comment:
Its important for our error messages to tell users what property failed to serialize. Can we update the code so that we continue to tell the user what property failed to serialize?

Reply:
The full error message is currently this:
```
jakarta.json.bind.JsonbException: Error accessing field: public int org.eclipse.yasson.defaultmapping.modifiers.ClassModifiersTest$NestedPackageParent.id
java.lang.IllegalAccessException: class is not public: org.eclipse.yasson.defaultmapping.modifiers.ClassModifiersTest$NestedPackageParent.id/int/getField, from org.eclipse.yasson.internal.model.PropertyModel (unnamed module @6da00fb9)
```
So the "Unable to serialize property 'id'" part from the test became the "Error accessing field: public int ..." part in the current version. This error is getting generated in "createReadHandle" or in "createWriteHandle" (we generate this error when we try to create the handle, so we never get to invoke the handle where the old message would be printed)

Reply:
@aguibert 
I updated the tests and the error message format